### ISSUE TITLE
Missing IAM permissions to run Metaflow commands from SageMaker notebook

### ISSUE THREAD

**Issue by mattmcclean** [Netflix/metaflow-tools#24]
I was attempting to use the SageMaker notebook instance configured as part of the Metaflow setup on AWS to run the Metaflow tutorial commands however it lacked the permissions to call AWS Batch and Step Function API calls. 

Not sure if this is by design as they are all supposed to be run from a local machine

**Comment by savingoyal** [comment]
@mattmcclean Yes, the tutorial is supposed to be run from a local machine. It's good feedback that we should look into allowing the execution from the Sagemaker notebook instance. The current cloud formation template is designed to provide a starting point for your Metaflow deployment. @queueburt FYI.
